packages/manager/modules/exchange/src/account/add/translations/Messages_fr_FR.json

```diff
@@ -1,3 +1,262 @@
 {
-  "exchange_account_add": "Ajout d'un compte"
+  "exchange_account_add": "Ajout d'un compte",
+  "exchange_account_country_AD": "Andorre",
```

Comment:
I think we might have already translated this for signup. This is quite painful to maintain, maybe we could think of a way we can share this list of translations

Reply:
Yes i have same opinion.
Regarding my tasks and time that i dedicate to develop it, i think i will schedule this refactor for another free time.

I will create a task to keep a trace.
Thanks.